tests/migrations/test_commands.py

```diff
@@ -413,6 +414,59 @@ def test_regression_22823_unmigrated_fk_to_migrated_model(self):
         """
         call_command("migrate", "migrated_unapplied_app", stdout=six.StringIO())
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_migrate_record_replaced(self):
+        """
+        Running a single squashed migration should record all of the original
+        replaced migrations as run.
+        """
+        recorder = MigrationRecorder(connection)
+        out = six.StringIO()
+        call_command("migrate", "migrations", verbosity=0)
+        call_command("migrate", "migrations", list=True, stdout=out, no_color=True)
```

Comment:
Use `showmigrations` instead of `"migrate", list=True` which is deprecated (we seem to have a bug in `@ignore_warnings` that causes it to leak state as the tests should catch the use of deprecated features and fail).